### PR TITLE
Add prettier-eslint to repo

### PR DESCRIPTION
**Pull request by billsaysthis**

### CODE REVIEW

**Comment by billsaysthis** [comment]
@clementallen Thanks for the suggestion, let me know if this works better for you.

File: src/sumologic.logger.min.js [hunk 1]
```diff
@@ -1 +1,208 @@
...d:c.session,url:encodeURI(e.location),timestamp:c.timestamp};return Object.keys(t).forEach(function(e){o[e]=t[e]}),JSON.stringify(o)});u=u.concat(h),0===this.interval&&this.sendLogs()},logSent:function(){},sendLogs:function(){if(0!==u.length){var e;try{var t=new XMLHttpRequest;e=u,u=[];var o=this.onError;if(t.addEventListener("error",function(){u=e,o&&o()}),this.onSuccess){var s=this.onSuccess;t.addEventListener("load",function(){s()})}t.open("POST",this.endpoint+"?callback=logSent",!0),t.setRequestHeader("Content-Type","application/json"),""!==this.name&&t.setRequestHeader("X-Sumo-Name",this.name),""!==this.sourceCategory&&t.setRequestHeader("X-Sumo-Category",this.sourceCategory),""!==this.hostName&&t.setRequestHeader("X-Sumo-Host",this.hostName),t.send(e.join("\n"))}catch(n){u=e,this.onError&&this.onError()}}},flushLogs:function(){this.sendLogs()},emptyLogQueue:function(){u=[]},getCookie:function(){var e=t.cookie,o=e.indexOf(r);if(0>o)return!1;var s=e.indexOf(";",o+1);return s=0>s?e.length:s,e.slice(o+a,s)},setCookie:function(e){t.cookie=r+"="+e}};var c=e.SLLogger,h=new o;if(c&&c.length){var g=0,f=c.length;for(g=0;f>g;g++)h.log(c[g])}e.SLLogger=h,e.SumoLogger=o}(window,document);
```
Comment (clementallen) [inline src/sumologic.logger.min.js]: Surely we don't want to prettify a minified file? :)

Comment (billsaysthis) [reply src/sumologic.logger.min.js]: I didn't intentionally do this...

Comment (billsaysthis) [reply src/sumologic.logger.min.js]: Updated

**Comment by clementallen** [comment]
LGTM! 👍

**Comment by billsaysthis** [comment]
Published as v2.4.0

**Pull request merged**